isis/tests/BundleSettingsTests.cpp

```diff
@@ -0,0 +1,825 @@
+#include "BundleSettings.h"
+
+#include <algorithm>
+
+#include <QBuffer>
+#include <QByteArray>
+#include <QString>
+#include <QList>
+#include <QPair>
+#include <QDomDocument>
+#include <QXmlStreamWriter>
+
+#include "BundleObservationSolveSettings.h"
+#include "BundleTargetBody.h"
+#include "IException.h"
+#include "MaximumLikelihoodWFunctions.h"
+
+#include "gmock/gmock.h"
+
+bool observationSettingsComparison(
+      BundleObservationSolveSettings m,
+      BundleObservationSolveSettings n
+) {
+  return (m.instrumentId() == n.instrumentId());
```

Comment:
Is there a reason that we are not using  EXPECT_TRUE(m.instrumentId() == n.instrumentId())  inside a TEST call...?

Reply:
This is used as a custom comparison between BundleObservationSolveSettings. It needs to just return a boolean in order for other things to handle the assertion and outputting error messages.